src/__tests__/articleTests/createArticle.test.js

```diff
@@ -92,3 +92,20 @@ it("test navbar component", () => {
   var node = component.find("Navbar");
   expect(node.length).toEqual(1);
 });
+
+it("should render my article component", () => {
+  const props = { dispatch: () => jest.fn() };
+  let event = {
+    target: {},
+    preventDefault: () => {
+      jest.fn();
+    }
+  };
+  const component = shallow(<CreateArticle {...props} />);
+  component.instance().handleChange(event);
+  component.instance().handleSelectChange(event);
+  component.instance().handleSubmit(event);
+  component.instance().handleEditorChange();
+  component.instance().categories();
```

Comment:
expect something to happen i.e   `expect(component.instance().handleChange({ preventDefault }));`

Reply:
Updated the test